call-home/src/bin/callhome/collector/report_models.rs

```diff
@@ -116,6 +200,166 @@ impl Nexus {
     }
 }
 
+/// StorageMedia contains storage media devices count, total capacity of all the storage media
+/// devices and disk_types which contains capacity and count of storage media for each disk_type.
+#[derive(Serialize, Deserialize, Debug, Default)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct StorageMedia {
+    count: u64,
+    total_capacity_in_bytes: u64,
+    disk_types: HashMap<String, DiskTypeInfo>,
```

Comment:
what is the key here?

Reply:
The key here is disk type (ex: "SSD") that we get from get_disk_type() function.